plugins/aiops/pkg/apis/modeltraining/modeltraining.proto

```diff
@@ -13,6 +13,12 @@ service ModelTraining {
       body: "*"
     };
   }
+  rpc PostModelTrainingStatus(ModelTrainingStatistics) returns (ModelTrainingResponse) {
+    option (google.api.http) = {
+      post: "/model/statistics"
```

Comment:
As this is calling a PUT on the kv (i.e. it is idempotently updating information) it should also be a PUT in the REST API, not a POST.